common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -104,4 +104,13 @@ trait ABTestSwitches {
     sellByDate = Some(LocalDate.of(2023, 2, 1)),
     exposeClientSide = true,
   )
+  Switch(
+    ABTests,
+    "ab-sign-in-gate-copy-test",
+    "Test the impact of changing the copy in the sign in gate",
+    owners = Seq(Owner.withEmail("personalisation.dev@theguardian.com")),
+    safeState = Off,
+    sellByDate = Some(LocalDate.of(2023, 2, 1)),
```

Comment:
it looks like the `sellByDate` is set for tomorrow, is that intentional?

Reply:
I usually set the sell by date with generous gap after the anticipated end of the test. I believe the sell by date kills the switch and therefore the test. 

We should probably raise a PR to amend the Main sign in gate switches to `sellByDate = never`

Reply:
Updated this to the end of may